smsapi/Api/Action/Contacts/CreateGroup.cs

```diff
@@ -3,58 +3,54 @@
 
 namespace SMSApi.Api.Action
 {
-    public class CreateGroup : Rest<Group>
+    public class CreateGroup : Base<Group>
     {
-        public string Description;
-
-        public string Idx;
-
-        public string Name;
-
-        protected override RequestMethod Method => RequestMethod.POST;
-
-        protected override NameValueCollection Parameters
-        {
-            get
-            {
-                NameValueCollection parameters = base.Parameters;
-                if (Name != null)
-                {
-                    parameters.Add("name", Name);
-                }
-
-                if (Description != null)
-                {
-                    parameters.Add("desciption", Description);
-                }
-
-                if (Idx != null)
-                {
-                    parameters.Add("idx", Idx);
-                }
-
-                return parameters;
-            }
-        }
-
-        protected override string Resource => "contacts/groups";
+        private string description;
```

Comment:
Here also you should not change access modifier if you proposed 1.1.0 version.

Reply:
I think we should go with that 2.0.0 as you proposed ;) because removing nullables will also introduce BC.